app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,53 @@
+class LikesController < ApplicationController
+  before_action :authenticate_user!
+  before_action :find_like, only: [:destroy]
+
+
+
+  def create
+    session[:return_to] ||= request.referer
+    @post = Post.find(params[:post_id])
+    @post.likes.create(user: current_user, post: @post)
+    @like = @post.likes.where(user: current_user, post: @post).first
```

Comment:
The result of the above statement is same as below.

Reply:
Done